app/jobs/import_versions_job.rb

```diff
@@ -69,6 +71,12 @@ def import_record(record)
       return
     end
 
+    if page.uuid_previously_changed?
+      log(object: page, operation: :created, row: row)
+    else
+      log(object: page, operation: :found, row: row, at: Time.current) # TODO: is it ok to use :found because the page is not updated
```

Comment:
I think it’s OK not to log this situation — my intuition is that it would be more noisy than helpful when trying to decipher what led to a failure from the logs. Do you think it’s critical information to keep around or would be especially helpful?

Reply:
I honestly don't really know what the shape of behavior is for imports (e.g. what the ratio is to pages and versions and whether it's more likely for a page to be newly created or already existing). 

But, from purely reading the complexity of code around whether to find or create a new page, it seems extra important to me to note what happened :-)